Public/EmailValidator.html

```diff
@@ -1,17 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="../assets/css/EmailValidator.css">
+
+    <title>Email validator</title>
+
     <title>Email Validator</title>
+
 </head>
+
 <body>
-    <form action="#">
-        Email <input type="text" name="name" id="name"><br>
+    <div class="container">
+        <h2> Email Validator
+            <hr>
+        </h2>
+        <p class="EmailId">
+            <label for="email">Email Id:</label>
+        </p>
+        <input type="email" id="email" placeholder="Enter your Email Id" required>
+        <div>
+            <button class="btn" id="submit"> Confirm Email </button>
+        </div>
+    </div>
+    <!-- <form action="#">
+        <label></label>
+       <input type="text" name="name" id="name"><br>
         <button id="submit"> Submit </button>
-    </form>
+    </form> -->
```

Comment:
There is no need for comments that are not related to the email validator. Please remove them if they do not serve any purpose.

Reply:
I deleted unwanted comments.